tests/sentry/lang/javascript/test_plugin.py

```diff
@@ -10,7 +10,9 @@
 from sentry.models import Event, File, Release, ReleaseFile
 from sentry.testutils import TestCase
 
-BASE64_SOURCEMAP = 'data:application/json;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZ2VuZXJhdGVkLmpzIiwic291cmNlcyI6WyIvdGVzdC5qcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiO0FBQUEiLCJzb3VyY2VzQ29udGVudCI6WyJjb25zb2xlLmxvZyhcImhlbGxvLCBXb3JsZCFcIikiXX0='
```

Comment:
Sort of unrelated change, but the sourcemap that was base64 encoded here was actually malformed, and our python implementation happened to work with it. But the new rust implementation fails against this fixture correctly. So this change changes the map to be correct, which allows both parsers to pass.